slack-api-model/src/main/java/com/slack/api/model/event/ChannelTopicMessageEvent.java

```diff
@@ -0,0 +1,26 @@
+package com.slack.api.model.event;
+
+import lombok.Data;
+
+/**
+ * https://api.slack.com/events/message/channel_topic
+ */
+@Data
+public class ChannelTopicMessageEvent implements Event {

```

Comment:
For subtyped events, we have a naming convention - `Message{subtype}Event`. Can you change the class name?
<img width="600" src="https://user-images.githubusercontent.com/19658/99864424-880da480-2be6-11eb-80ad-15205b4c78d2.png">


```suggestion
public class MessageChannelTopicEvent implements Event {

```